### PR TITLE
fixes Issue 251 (future warning)

### PR DESCRIPTION
**Pull request by fangohr**
Commit  a8be825 addresses the issue.

### CODE REVIEW

File: oscovida/oscovida.py [hunk 1]
```diff
@@ -144,7 +144,8 @@ def get_country_data_johns_hopkins(country: str,
     if len(tmp.shape) == 1:
         d = deaths.loc[country]
     elif len(tmp.shape) == 2:   # China, France, United Kingdom, ...
-        d = deaths.loc[country].sum()
+        tmp2 = tmp.drop(columns=['Province/State'])
```
Comment (kirienko) [inline oscovida/oscovida.py:147]: Do we need this intermediate `tmp2` object?

**Review by kirienko — COMMENTED** [review]
I don't actually understand the problem here. Could you explain? :man_shrugging:

File: oscovida/oscovida.py (hunk 1 above)
Comment (fangohr) [reply oscovida/oscovida.py]: Good point. The next commits try to tidy this up.

Comment (fangohr) [reply oscovida/oscovida.py]: > I don't actually understand the problem here. Could you explain? 🤷‍♂️

The table has a column "Province/State" which contains strings. The other columns are one for each date, and one each for longitude and latitude: all carry numbers. Thus ``sum`` works well on all columns apart from "Province/State". That's the one we need to remove (before calling ``sum``) to get rid of the new deprecation warning from pandas.

Comment (kirienko) [reply oscovida/oscovida.py]: Thanks for the explanation!

**Review by kirienko — APPROVED** [review]
LGTM!

**Pull request merged**